tox.ini

```diff
@@ -1,6 +1,5 @@
 [tox]
 envlist =
-    py35
```

Comment:
Please also adapt the `classifiers` and `python_requires` in `setup.cfg` accordingly.